aspnetcore/mvc/controllers/actions.md

```diff
@@ -14,80 +14,86 @@ uid: mvc/controllers/actions
+* **HTTP Status Code**
 
-**Content negotiated response**
+    This type returns an HTTP status code. A couple helper methods of this type are `BadRequest` and `NotFound`. For example, `return BadRequest();` produces a 400 status code when executed.
 
-Instead of returning an object directly, an action can return a content negotiated response (using `Ok`, `Created`, `CreatedAtRoute` or `CreatedAtAction`). Examples: `return Ok();` or `return CreatedAtRoute("routename",values,newobject);`
+#### 2. A non-empty response body with a predefined content type
 
-**Redirect**
+Most of these response types include a `ContentType` property, allowing you to set the `Content-Type` response header to describe the response body.
 
-Returns a redirect to another action or destination (using `Redirect`, `LocalRedirect`, `RedirectToAction` or `RedirectToRoute`). Example: `return RedirectToAction("Complete", new {id = 123});`
+There are two response types within this category: [View](xref:mvc/views/overview) and [Formatted Response](xref:mvc/models/formatting).
 
-In addition to the methods above, an action can also simply return an object. In this case, the object will be formatted based on the client's request. Learn more about [Formatting Response Data](../models/formatting.md)
+* **View**
 
-### Cross-Cutting Concerns
+    This type returns a view which uses a model to render HTML. For example, `return View(customer);` passes a model to the view for data-binding.
+
+* **Formatted Response**
+
+    This type returns JSON or a similar data exchange format to represent an object in a specific manner. For example, `return Json(customer);` serializes the provided object into JSON format.
 
-In most apps, many actions will share parts of their workflow. For instance, most of an app might be available only to authenticated users, or might benefit from caching. When you want to perform some logic before or after an action method runs, you can use a *filter*. You can help keep your actions from growing too large by using [Filters](filters.md) to handle these cross-cutting concerns. This can help eliminate duplication within your actions, allowing them to follow the [Don't Repeat Yourself (DRY) principle](http://deviq.com/don-t-repeat-yourself/).
+#### 3. A non-empty response body formatted in a content type negotiated with the client
+
+This type is better known as a **Content Negotiated Response**. [Content negotiation](xref:mvc/models/formatting#content-negotiation) applies whenever an action returns an [ObjectResult](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.mvc.objectresult) type or something other than an [IActionResult](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.mvc.iactionresult) implementation. 
+
+Some helper methods of this type include `Created`, `CreatedAtAction`, `CreatedAtRoute`, and `Ok`. Examples: `return Ok();` or `return CreatedAtRoute("routename", values, newobject);`
```

Comment:
As mentioned above, `Ok()` without parameters is just a status code responder. `Created(...)` is a good example because its overloads all include a `value`.

Suggest mentioning `BadRequest(ModelStateDictionary modelState)` because it seems to get lots of use.